### PR TITLE
Tools were written to expect the edit command to

### PR DESCRIPTION
**Pull request by bbastings**
return the command string from onStart.

### CODE REVIEW

File: common/changes/@itwin/editor-frontend/fix-builtin-edit-commands_2022-09-01-16-21.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/editor-frontend",
+      "comment": "Fix spelling in method docs.",
```
Comment (aruniverse) [inline common/changes/@itwin/editor-frontend/fix-builtin-edit-commands_2022-09-01-16-21.json:5]:
```suggestion
      "comment": "",
```

**Review by aruniverse — APPROVED** [review]

**Review by pmconne — APPROVED** [review]

File: common/changes/@itwin/editor-frontend/fix-builtin-edit-commands_2022-09-01-16-21.json (hunk 1 above)
Comment (bbastings) [reply common/changes/@itwin/editor-frontend/fix-builtin-edit-commands_2022-09-01-16-21.json:5]: Why is this important? If we aren't supposed to describe the change for "rush change" make it not ask me?

Comment (aruniverse) [reply common/changes/@itwin/editor-frontend/fix-builtin-edit-commands_2022-09-01-16-21.json:5]: This is just a nit; feel free to ignore.

The rush change is used to generate the changelog. imo i dont think we should care about spelling changes or anything non api related that we want to explicitly call out

Comment (bbastings) [reply common/changes/@itwin/editor-frontend/fix-builtin-edit-commands_2022-09-01-16-21.json:5]: Okay, I will keep that rule in mind for the future,

**Pull request merged**